lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2637,6 +2637,10 @@ static ArrayRef<Decl *> evaluateMembersRequest(
         (void) var->getPropertyWrapperAuxiliaryVariables();
         (void) var->getPropertyWrapperInitializerInfo();
       }
+
+      if (auto *func = dyn_cast<FuncDecl>(member)) {
+        (void) func->getDistributedActorRemoteFuncDecl();
+      }
```

Comment:
so basically we need to trigger synthesis/"get" in same spots as property wrappers most of the time -- one is here, and another is in lookups

Reply:
(The lookups solve the across files case.)